go-controller/pkg/ovn/defaultcontroller.go

```diff
@@ -0,0 +1,845 @@
+		np := obj.(*knet.NetworkPolicy)
+		klog.V(5).Infof("Recording add event on network policy %s/%s", np.Namespace, np.Name)
+		metrics.GetConfigDurationRecorder().Start("networkpolicy", np.Namespace, np.Name)
+	}
+}
+
+// RecordUpdateEvent records the update event on this given object.
+func (h *defaultNetworkControllerEventHandler) RecordUpdateEvent(obj interface{}) {
+	switch h.objType {
+	case factory.PodType:
+		pod := obj.(*kapi.Pod)
+		klog.V(5).Infof("Recording update event on pod %s/%s", pod.Namespace, pod.Name)
+		metrics.GetConfigDurationRecorder().Start("pod", pod.Namespace, pod.Name)
+	case factory.PolicyType:
+		np := obj.(*knet.NetworkPolicy)
+		klog.V(5).Infof("Recording update event on network policy %s/%s", np.Namespace, np.Name)
+		metrics.GetConfigDurationRecorder().Start("networkpolicy", np.Namespace, np.Name)
+	}
+}
+
+// RecordDeleteEvent records the delete event on this given object.
+func (h *defaultNetworkControllerEventHandler) RecordDeleteEvent(obj interface{}) {
+	switch h.objType {
+	case factory.PodType:
+		pod := obj.(*kapi.Pod)
+		klog.V(5).Infof("Recording delete event on pod %s/%s", pod.Namespace, pod.Name)
+		h.oc.podRecorder.CleanPod(pod.UID)
+		metrics.GetConfigDurationRecorder().Start("pod", pod.Namespace, pod.Name)
+	case factory.PolicyType:
+		np := obj.(*knet.NetworkPolicy)
+		klog.V(5).Infof("Recording delete event on network policy %s/%s", np.Namespace, np.Name)
+		metrics.GetConfigDurationRecorder().Start("networkpolicy", np.Namespace, np.Name)
+	}
+}
+
+// RecordSuccessEvent records the success event on this given object.
+func (h *defaultNetworkControllerEventHandler) RecordSuccessEvent(obj interface{}) {
+	switch h.objType {
+	case factory.PodType:
+		pod := obj.(*kapi.Pod)
+		klog.V(5).Infof("Recording success event on pod %s/%s", pod.Namespace, pod.Name)
+		metrics.GetConfigDurationRecorder().End("pod", pod.Namespace, pod.Name)
+	case factory.PolicyType:
+		np := obj.(*knet.NetworkPolicy)
+		klog.V(5).Infof("Recording success event on network policy %s/%s", np.Namespace, np.Name)
+		metrics.GetConfigDurationRecorder().End("networkpolicy", np.Namespace, np.Name)
+	}
+}
+
+// RecordErrorEvent records an error event on the given object.
+// Only used for pods now.
+func (h *defaultNetworkControllerEventHandler) RecordErrorEvent(obj interface{}, reason string, err error) {
+	switch h.objType {
+	case factory.PodType:
+		pod := obj.(*kapi.Pod)
+		klog.V(5).Infof("Recording error event on pod %s/%s", pod.Namespace, pod.Name)
+		h.oc.recordPodEvent(reason, err, pod)
+	}
+}
+
+// IsResourceScheduled returns true if the given object has been scheduled.
+// Only applied to pods for now. Returns true for all other types.
+func (h *defaultNetworkControllerEventHandler) IsResourceScheduled(obj interface{}) bool {
+	return h.masterEventHandler.isResourceScheduled(h.objType, obj)
+}
+
+// Given a *RetryFramework instance, an object to add and a boolean specifying if the function was executed from
```

Comment:
nit: I know you just moved this code, but this commetn should start with // AddResource. If you happen to push another change then you can just update it, otherwise dont worry about it.

Reply:
will update it in the subsequent PRs if no other changes needed for this PR.